framework/src/kernels/MatBodyForce.C

```diff
@@ -0,0 +1,71 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MatBodyForce.h"
+#include "Function.h"
```

Comment:
Doesn't seem to be used (and neither is the forward declaration in the header)

Reply:
Yeah, copy paste error